src/neuroconv/datainterfaces/behavior/neuralynx/neuralynx_nvt_interface.py

```diff
@@ -0,0 +1,142 @@
+import json
+from typing import Optional
+
+import numpy as np
+from pynwb import NWBFile
+from pynwb.behavior import CompassDirection, Position, SpatialSeries
+
+from .nvt_utils import read_data, read_header
+from ....basetemporalalignmentinterface import BaseTemporalAlignmentInterface
+from ....utils import DeepDict, FilePathType, NWBMetaDataEncoder, get_base_schema
+
+
+class NeuralynxNvtInterface(BaseTemporalAlignmentInterface):
+    """Data interface for Neuralynx NVT files. NVT files store position tracking information"""
+
+    def __init__(self, file_path: FilePathType, verbose: bool = True):
+        """
+        Interface for writing Neuralynx .nvt files to nwb.
+
+        Parameters
+        ----------
+        file_path : FilePathType
+            Path to the .nvt file
+        verbose : bool, default: True
+            controls verbosity.
+        """
+
+        self.file_path = file_path
+        self.verbose = verbose
+        self._timestamps = self.get_original_timestamps()
+        self.header = read_header(self.file_path)
+        self.nvt_filename = self.header["OriginalFileName"].split("\\")[-1]
```

Comment:
Definitely a good idea to use the filename below as the metadata reference key; one minor suggestion would be to strip the suffix and just use the stem

Also suggest using `Path` to accomplish this; the example data was clearly from a Windows system, but if it's possible for it to be from any other OS then the string splitting method might not have worked as expected

```suggestion
        self.nvt_filename = Path(self.header["OriginalFileName"]).stem
```

Reply:
It was trickier than I thought to parse filepaths from different operating systems, so I created a little utility function. I'd rather keep the suffix so it's clear to the user that this is from the filepath